app/models/user.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  return sequelize.define(
+    'User',
```

Comment:
you should use the same name here and in the migration. Also, it should start with lower case because of the postgres convention (I'm almost sure)